schematics/src/lazy-component/factory.ts

```diff
@@ -108,12 +121,28 @@ export function createLazyComponent(options: Options): Rule {
         }
       }
     }
+    const exportsModuleName = `${isProject ? projectName : extension}-exports`;
+    const exportsModuleExists = host.exists(`${options.path}/${exportsModuleName}.module.ts`);
 
     const operations = [];
 
     if (!options.ci) {
+      if (!exportsModuleExists) {
```

Comment:
In the same fashion: if the exports folder does not contain a `.gitignore` with `/lazy**`, it should be added, because generated lazy components should not be committed to the source repository.